ci - drone failures on master branch because of docs

the past few commits have been failing on master due to the ghpages make recipe failing

```
$ make ghpages
git checkout gh-pages && \
mv docs/build/html new-docs && \
rm -rf docs && \
mv new-docs docs && \
git add -u && \
git add -A && \
git commit -m "Updated generated Sphinx documentation"
error: The following untracked working tree files would be overwritten by checkout:
	.pytest_cache/README.md
	tools/c7n_azure/tests/cassettes/TagsTest.test_tag_trim_space_must_be_btwn_0_and_15.yaml
Please move or remove them before you can switch branches.
Aborting
Makefile:47: recipe for target 'ghpages' failed
make: [ghpages] Error 1 (ignored)
$ git branch
* master
$ git checkout master
Already on 'master'
[info] Pulling image plugins/drone-git-push:latest
Drone Git Push Plugin built from e1d0995
$ git push origin gh-pages:gh-pages
error: src refspec gh-pages does not match any.
error: failed to push some refs to 'https://github.com/capitalone/cloud-custodian.git'
exit status 1
[info] build failed (exit code 1)
```

--- a/.gitignore
+++ b/.gitignore
@@ -28,3 +28,4 @@ results.xml
 .idea/
 .vscode/
 !.vscode/*.sample
+.pytest_cache/

